Automattic/wp-calypso
# P2: Don't show privacy settings for hubs

#### Changes proposed in this Pull Request

This PR hides the privacy settings for P2 hubs because they don't apply.

#### Testing instructions

Test on a url like this: http://calypso.localhost:3000/settings/general/[path-to-site]

* On a site that is not a hub (P2 or not does not matter), verify that you get the privacy settings here: 
<img width="769" alt="Screen Shot 2021-05-31 at 12 56 51" src="https://user-images.githubusercontent.com/193283/120183201-ae547280-c20f-11eb-9e0b-dc59bf84a97b.png">

* On a hub, verify that you don't get the privacy settings from the screenshot above




See `2947-gh-Automattic/p2`
